### PR TITLE
Add Red Hat support

### PR DESCRIPTION
**Pull request by alexjurkiewicz**

### CODE REVIEW

File: manifests/check.pp [hunk 1]
```diff
@@ -49,6 +49,7 @@
   $package_name = 'monit',
   $service_name = 'monit',
 ) {
+  include monit::params
```
Comment (alexjurkiewicz) [inline manifests/check.pp]: This is the only line I'm not sure you'll be happy with. Is this how you want to expand monit::params into the type?

**Comment by alexjurkiewicz** [comment]
ping

File: manifests/check.pp (hunk 1 above)
Comment (floSoX) [inline manifests/check.pp]: This line is compliant with the best practices formulated by Puppet Labs : https://docs.puppetlabs.com/puppet/latest/reference/lang_defined_types.html#defining-a-type.
So I am happy with this.

**Comment by floSoX** [comment]
Thank you for your work!
I will work to merge your work soon.
Probably next week.
Regards,

**Comment by aamerik** [comment]
Hi, thank you for this module. Can you PLEASE merge this PR? We need RHEL support big time.

Thank you

**Comment by ctohastek** [comment]
Tried to use the module - no go - unsupported OS Family RedHat - I think the issue is the operatingsystemrelease fact - my boxes = 6.6 --- the module allows for only '5.0' and '6.0' !!

File: metadata.json [hunk 2]
```diff
@@ -14,6 +14,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
```
Comment (aamerik) [inline metadata.json]: Will this work for RHEL 6.5, 6.6. etc? Perhaps a better fact to use is "operatingsystemmajrelease"

Comment (floSoX) [inline metadata.json]: As you can see here : https://github.com/puppetlabs/puppetlabs-ntp/blob/master/metadata.json
You can keep the 'operatingsystemrelease' keyword but you have to remove '.0' in this versions

Comment (alexjurkiewicz) [inline metadata.json]: Hi all,
I'm not interested in this PR any more, so please feel free to take it over.

**Comment by floSoX** [comment]
After investigation, I realize that Monit is not in official RedHat/CentOS repositories.
So to really support RedHat OS Family, we should add the EPEL repositories to yum (probably with this module: https://forge.puppetlabs.com/stahnma/epel).
Anyway, I don't know the best practices to do so, a line in the documentation could be enough. If you have any suggestions or if you can patch this, it will be a great help.

**Comment by aamerik** [comment]
@floSoX For now would it be possible to add the case statement for RedHat and implement optional EPEL support down the road?

Thanks

**Comment by floSoX** [comment]
@aamerik I did it yesterday.
Thank you for your interest in our module.
Regards,

**Pull request closed without merging**